src/python/pants/backend/cue/goals/fix.py

```diff
@@ -30,8 +33,8 @@ async def run_cue_fmt(request: CueFmtRequest.Batch, cue: Cue, platform: Platform
     return await FmtResult.create(request, process_result)
 
 
-def rules():
-    return [
+def rules() -> Iterable[Rule]:
+    return (
```

Comment:
Each of these files is switching from list to tuple, and adding the return annotation. I don't see a problem with doing that, though I'm curious about your reasoning: why make this migration? Just trying to make everything consistent? (And, how did mypy not complain about the lack of type hints?)

Reply:
Yeah, just trying to mildly update to any amount of consistency - since I'm already in the files, and we use tuples more than lists.

Adding the type hints where possible - because Pants is severely under typed. There are so many type check problems, I don't know why mypy doesn't go crazier. In the `rules` case, it doesn't care - but I view `def rules` as basically an external API for the file, which should definitely be typed.